content/client-lib-development-guide/features.textile

```diff
@@ -1332,6 +1335,16 @@ h4. ChannelOptions
 ** @(TB2b)@ @cipher@, which is either:
 *** @(TB2b1)@ A @CipherParams@ instance, or
 *** @(TB2b2)@ an options hash (or language equivalent) consisting of any subset of @CipherParams@ fields that includes a @key@. In this case, the client library should call @getDefaultParams@, passing it the options hash, to obtain a @CipherParams@ instance
+*** @(TB2c)@ @params@ If provided these are sent to Ably in a new params property @(TR4n)@ of the ATTACH ProtocolMessage. Realtime will echo back the applied params in the ATTACHED if that field was specified in the ATTACH. Params could be:
```

Comment:
>  If provided 

If provided by what.  This is a data structure.

> these are sent to Ably in a new params property @(TR4n)@ of the ATTACH ProtocolMessage

I would think "...a new params property" will soon just become "the @params@ property".  We should not be referencing things as new in the spec.

> Realtime will echo back the applied params in the ATTACHED if that field was specified in the ATTACH.

This spec should not in `ChannelOptions`, whihc is just a datatype.

Under what conditions is an `ATTACH` message sent, and when will it be sent back. Find where that happens in the spec, and then developers will understand under what condition this happens, and how to test it. 

Please note in Textile we use `@` symbol to surround code blocks. This is all covered in the [documentation style guide](https://docs.ably.io/client-lib-development-guide/documentation-formatting-guide/) in examples through this document.

Reply:
At the time of your review this should not have been part of this PR any more. I removed this spec completely and left it to https://github.com/ably/docs/pull/766 to document channel params. You must have been looking at some older version of this PR.